Backend/index.js

```diff
@@ -1,32 +0,0 @@
-const express = require("express");
-const cors = require('cors');
-const connection = require("./Config/conn");
-const {UserController, GoogleController} = require("./Routes/AutenticationRoutes/user.route");
-require('dotenv').config()
-const TaskController  = require('./Routes/AppRoutes/Task.route');
-
-const authentication = require("./Middleweres/Authentication")
-const TimeController  = require('./Routes/AppRoutes/Time.route')
-
-
-const app = express();
-app.use(cors())
```

Comment:
> var corsOptions = {
    origin: 'http://localhost:8080',
    optionsSuccessStatus: 200 // For legacy browser support
}
app.use(cors(corsOptions));`


cors only allow required sites, not all sites